ScoutSuite/providers/azure/rules/findings/securitycenter-auto-provisioning-off.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dashboard_name": "Security Center",
+    "description": "Automatic provisioning of monitoring agent",
+    "rationale": "You should set automatic provisioning of monitoring agent. See CIS 2.2.",
```

Comment:
In GCP, we specify the CIS rules with parantheses (ie. `Database servers should accept connections from trusted IPs and networks only (CIS 6.2).`). I think it would be preferable to be consistent with that as well.